docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
Thanks @alisd23 - agreed it's quite a long one. 

I really like the `daemon` suggestion by @istarkov.  I was actually on my way here to suggest an alternative of `epic`, but the `daemon` suggestion seems more intuitive.